_posts/2021-02-20-elixir-aai-login.md

```diff
@@ -0,0 +1,51 @@
+---
+site: freiburg
+tags: [elixir]
```

Comment:
we haven't used this tag so far... What about something more generic like 'FAQ'? We could add that to the [EU flavours](https://galaxyproject.eu/posts/2020/12/28/subdomains/), [remote training](https://galaxyproject.eu/posts/2020/09/15/ML-lessons-learned/) or the [3 steps to get your tool into Galaxy](https://galaxyproject.eu/posts/2020/08/22/three-steps-to-galaxify-your-tool/)

Reply:
Ok. Like it